src/server/GRPCServerEndPoint.cpp

```diff
@@ -68,10 +86,15 @@ grpc::Status GRPCServerEndPoint::RemoveTaskLabel(::grpc::ServerContext *context,
 grpc::Status GRPCServerEndPoint::GetTasks(::grpc::ServerContext *context,
                                              const ::GetTasksRequest *request,
                                              ::GetTasksResponse *response) {
+    LOG_TRIVIAL(debug) << "Request received";
```

Comment:
what request?

Reply:
`LOG_TRIVIAL(severity_level)` is my custom log, that automatically adds file name (class name) and method name to log, so that you can know what request was received and where (`GetTasks` in `GRPCServerEndPoint`)